_pages/mobile-interview-project.md

```diff
@@ -193,17 +195,16 @@ Please do not use code from previous/current employers, for IP / legal reasons :
 
  * Q) **What programming languages can I use?**
 
-Please utilize Java or Kotlin for Android, or Objective-C or Swift for iOS. For iOS, if you need to drop down to CoreFoundation, Accelerate, etc, C is fine too (but you shouldn’t need to).
-
+Please utilize Java or Kotlin for Android, or Objective-C or Swift for iOS.
 Do not feel pressured or required to use Swift or Kotlin if you’ve primarily worked in Objective-C or Java in the past – use whatever you’re most comfortable with! For context, Square still has plenty of Objective-C and Java, and will for some time.
 
-Please do not utilize cross-platform SDKs such as React Native, Flutter, Phonegap, Cordova, etc. We do not utilize these at Square.
+For this project, please do not utilize cross-platform SDKs such as React Native, Flutter, Phonegap, Cordova, etc.
 
- * Q) **Can I use 3rd party frameworks?**
+ * Q) **Can I use 3rd party frameworks and libraries?**
 
-Sure – feel free to use any “no brainer” frameworks you’d usually pull into an app such as RxJava, RxSwift, etc. However, since this is a relatively simple application, you shouldn’t need many external dependencies (in general, Square avoids depending on too many external dependencies).
+Sure – feel free to use frameworks and libraries you’d consider essential in your application development process. These might include your preferred frameworks and libraries for handling dependency injection, network requests, JSON deserialization, etc. Note, however, that Square avoids unnecessary external dependencies - we recommend using as few external dependencies as is reasonably possible for this project.
 
-If you copy in dependencies (versus reference them with a dependency manager), please do note as such in your README.
+Also, If you copy 3rd party framework or library source files into your project, make sure that you attribute them in the relevant section of the README.
```

Comment:
I like the new wording!

Reply:
I like it, too, particularly, the first two new sentences ending with the examples! I think that really captures the point. 

What do you think about removing the "Note, however, that Square...". Who doesn't want to avoid _unnecessary_ external dependencies? They're unnecessary! :) And there's a sort of conflict between saying, sure, go ahead and use some, but hey, we don't recommend using _too_ many of them. I think the first part conveys the sorts of things we'd expect, particularly because of the examples and your pointing them at those they consider "essential". I like that. That should be enough to avoid confusion, and if they add some really oddball, unnecessary library after that, that's a bit of a signal. And I don't think we've tricked them into giving us that signal.

Reply:
I agree @mrubinsquare - that gets to the point more directly, and like you say we're suggesting our recommendation by using the word "essential". I'll drop it!